Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneDefinition/Components/SceneDefinitionComponent.cs

```diff
@@ -9,61 +8,130 @@
 namespace ECS.SceneLifeCycle.SceneDefinition
 {
     /// <summary>
-    ///     Scene definition in ECS, indicates the definition is loaded but does tell the state of SceneFacade itself
+    ///     Scene definition in ECS, indicates the definition is loaded but does not tell the state of SceneFacade itself
     /// </summary>
     public struct SceneDefinitionComponent
     {
-        public static readonly SceneMetadataScene EMPTY_METADATA = new ();
+        private static readonly SceneMetadataScene EMPTY_METADATA = new ();
```

Comment:
you forgot to move it to the factory to avoid allocations for every empty scene

Reply:
Good catch!